src/modules/components/proposal/proposalActions/proposalActionsAction/proposalActionsAction.test.tsx

```diff
@@ -73,4 +73,40 @@ describe('<ProposalActionsAction /> component', () => {
         await userEvent.click(screen.getByRole('button'));
         expect(screen.getByTestId(testId)).toBeInTheDocument();
     });
+
+    it('renders AlertCard when action.value is not null and not "0"', async () => {
```

Comment:
Let's keep it generic, also on the tests below
```suggestion
    it('renders an alert when action.value is not null and not "0"', async () => {
```